rclcpp_lifecycle/src/lifecycle_node_interface_impl.hpp

```diff
@@ -96,6 +96,7 @@ class LifecycleNode::LifecycleNodeInterfaceImpl
       rosidl_typesupport_cpp::get_service_type_support_handle<GetAvailableStatesSrv>(),
       rosidl_typesupport_cpp::get_service_type_support_handle<GetAvailableTransitionsSrv>(),
       rosidl_typesupport_cpp::get_service_type_support_handle<GetAvailableTransitionsSrv>(),
+      enable_communication_interface,
```

Comment:
if this is false, publication on topic named `/<node_name>/transition_event` will not be initialized. but rclcpp_lifecycle will try to publish `lifecycle_msgs__msg__TransitionEvent` once `change_state` is called. i may be missing something, but i thnink this will cause transition error.

Reply:
very well seen! I think it makes sense to enable the event publisher in any case and only disable the actual services.
I'll change that accordingly in the rcl scope.

Reply:
> I think it makes sense to enable the event publisher in any case and only disable the actual services.

So do I.